modular_skyrat/modules/mars_event/code/terrain.dm

```diff
@@ -0,0 +1,57 @@
+#define RED_PLANET_ATMOS "co2=40;o2=3;n2=7;TEMP=210"
+#define RED_ROCK "#934b33"
+#define DARK_ROCK "#2b2b2b"
+
+/turf/open/misc/ironsand/redplanet
+	initial_gas_mix = RED_PLANET_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/misc/asteroid/basalt/redplanet
+	baseturfs = /turf/baseturf_bottom
+	initial_gas_mix = RED_PLANET_ATMOS
+	planetary_atmos = TRUE
+
+/turf/closed/mineral/random/redplanet
+	baseturfs = /turf/baseturf_bottom
+	mineralChance = 25
+	mineralAmt = 10
+	color = RED_ROCK
+	mineralSpawnChanceList = list(
+		/obj/item/stack/ore/uranium = 5,
+		/obj/item/stack/ore/diamond = 1,
+		/obj/item/stack/ore/gold = 10,
+		/obj/item/stack/ore/silver = 12,
+		/obj/item/stack/ore/plasma = 20,
+		/obj/item/stack/ore/iron = 40,
+		/obj/item/stack/ore/titanium = 11,
+		/obj/item/stack/ore/bluespace_crystal = 1
```

Comment:
no bananium?

Reply:
this is default ore spawn with gibtonite removed so yes, no bananium